Include/internal/pycore_interp.h

```diff
@@ -259,7 +259,8 @@ struct _is {
     struct callable_cache callable_cache;
     _PyOptimizerObject *optimizer;
     _PyExecutorObject *executor_list_head;
-
+    // executors_created is limited to JIT_CLEANUP_THRESHOLD
+    size_t executors_created;
```

Comment:
I'd rename this to `new_executors` or something similarly descriptive. The current name makes it sound like a total count, rather than "executors created since the last cleanup".